src/ooxml-viewer.ts

```diff
@@ -215,12 +225,13 @@ export class OOXMLViewer {
         matchWholeWord: false,
       });
     } catch (err) {
-      if (err?.code === 'ENOENT' || err?.code === 'FileNotFound') {
+      if (
```

Comment:
That's true for vscode FileSystemError, but it could also be a Node FileSystem error, which uses code `ENOENT` for missing files. They both have a code property, so that's why I added the `Object.prototype.hasOwnProperty.call(err, 'code')`, to catch any error that has a code property, regardless of type. This also fixed a headache with the tests. `toLowerCase()` isn't strictly necessary, but I find it a good practice to toLowerCase() strings before comparison if case doesn't matter.

Reply:
If we're using `workspace.fs` methods, won't it only throw vscode FileSystemErrors?

Reply:
I was trying to be thorough, but you're right, `workspace.fs` won't throw `ENOENT`, I will remove the `ENOENT` check.